satpy/readers/aapp_l1b.py

```diff
@@ -568,6 +570,5 @@ def _ir_calibrate(header, data, irchn, calib_type, mask=False):
         tb_ = (t_planck - bandcor_2) / bandcor_3
 
     # Mask unnaturally low values
-    # mask |= tb_ < 0.1
 
-    return np.where(mask, np.nan, tb_)
+    return da.where(mask, tb_, np.nan)
```

Comment:
So the `mask` is the inverse of what it was?

Reply:
Yes it's been inversed